openmdao/core/driver.py

```diff
@@ -176,6 +176,7 @@ def __init__(self, **kwargs):
         self.supports.declare('active_set', types=bool, default=False)
         self.supports.declare('simultaneous_derivatives', types=bool, default=False)
         self.supports.declare('total_jac_sparsity', types=bool, default=False)
+        self.supports.declare('distributed_design_vars', types=bool, default=True)
```

Comment:
why default=True?  seems like default=False is safer and would not require all the existing drivers to set it

Reply:
I set that to True when I thought some of our distributed tests were testing derivs wrt distributed variables, but it turns out those were all remote. I will set this back to False (but keeping it in all the sub drivers for completeness.